P5/Source/Specs/move.xml

```diff
@@ -173,7 +104,7 @@ $Id$
         </castList>
       </performance>
       <!-- ... -->
-      <stage type="entrance"><move who="#bellaf" type="enter" where="L" perf="#perf1"/> Enter
+      <stage type="entrance"><move who="#bellaf" type="enter" where="#L" perf="#perf1"/> Enter
```

Comment:
I think we'll need an example for resolving the `where="#L"` as all other targets of references ("#bellaf" and "#perf1") are provided.